Mathlib/SetTheory/Ordinal/Notation.lean

```diff
@@ -60,7 +60,7 @@ instance : One ONote :=
   ⟨oadd 0 1 0⟩
 
 /-- Notation for ω -/
-def omega : ONote :=
+def omega0 : ONote :=
```

Comment:
I think we can save this change for later. `Notation` is a leaf file and it's unlikely to see much action anytime soon...